plugin/src/compiler.rs

```diff
@@ -272,14 +272,12 @@ fn compile_op(ecx: &ExtCtxt, buffer: &mut StmtBuffer, op: Ident, prefixes: Vec<I
             } else if op_size != Size::OWORD {
                 panic!("bad formatting data");
             }
-        } else {
-            if op_size == Size::WORD {
-                pref_size = true;
-            } else if op_size == Size::QWORD {
-                rex_w = true;
-            } else if op_size != Size::DWORD {
-                panic!("bad formatting data");
-            }
+        } else if op_size == Size::WORD {
+            pref_size = true;
+        } else if op_size == Size::QWORD {
+            rex_w = true;
+        } else if op_size != Size::DWORD {
+            panic!("bad formatting data");
```

Comment:
I'd like to keep this part within the else block to be consistent with the previous blocks. A better formatting solution would probably be to just match on op_size for each size encoding flag, it'd be shorter and more clear.